### ISSUE TITLE
HELP Please!!!! Image was not liked YOU HAVE A BLOCK ON LIKES

### ISSUE THREAD

**Issue by Khumoyun098** [InstaPy/InstaPy#5702]

**Comment by Khumoyun098** [comment]
I can't like and comment -------- WARNING! Image was NOT liked! You have a BLOCK on likes!

**Comment by JManke91** [comment]
That happens sometimes when IG detects a lot of actions going on in your account. Wait a few hours/days and when this temporary action block is gone, use `set_quota_supervisor` with low settings in order to control the number of likes, follows etc. your bot performs per hour/day.

**Comment by Khumoyun098** [comment]
SOLUTION:

1. OPEN browser.py file
2. Find user-agent dict
3. Change it knew 
4. Save

**Comment by itssmaxy** [comment]
> SOLUTION:
> 
> 1. OPEN browser.py file
> 2. Find user-agent dict
> 3. Change it knew
> 4. Save

Hey @Khumoyun098 what did you change user_agent to? 

EDIT: I found more firefox user_agents online. IS it enough just to change the string for user_agent or do i need to download a file and replace it with current agent?

EDIT2: Basically I got to change the user agent but it didn't change anything. Still restricted. I'll give it a few days and try again
